src/pages/namjanghyun/Main/Main.js

```diff
@@ -1,7 +1,37 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import "./Main.scss";
+import Feeds from "./components/Feeds";
+import Aside from "./components/Aside";
+import Nav from "../../../components/Nav/Nav";
 
 function MainNam() {
-  return <h1>Hello world</h1>;
+  const [feedDataList, setFeedDataList] = useState([]);
+
+  useEffect(() => {
+    fetch("./data/feedData.json")
+      .then((response) => response.json())
+      .then((result) => setFeedDataList(result));
+  }, []);
+
+  return (
+    <div className="containerWrapperMain">
+      <Nav />
```

Comment:
containerWrapperMain으로 nesting된 스타일링이 Nav 컴포넌트에도 적용될 수 있음
e.g. flexbox
```suggestion
    <>
      <Nav />
      <div className="containerWrapperMain">
```